### PR TITLE
a11y button fixes

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**
- Adds the role "image" to mode SVGs, to ensure the aria-labels aren't overlooked by AT
- Refactors "Copy Link" button `aria-live` to be more consistent across AT

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [n/a] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/narrative/trip-tools.js [hunk 1]
```diff
@@ -55,6 +56,12 @@ class CopyUrlButton extends Component {
   render() {
     return (
       <div>
+        {/* Announces copy button status to AT */}
+        <InvisibleA11yLabel aria-live="assertive">
+          {this.state.showCopied && (
+            <FormattedMessage id="components.TripTools.linkCopied" />
```
Comment (miles-grant-ibigroup) [inline lib/components/narrative/trip-tools.js:62]: The conditional should probably include the label too right

File: lib/components/narrative/trip-tools.js [hunk 2]
```diff
@@ -55,11 +56,13 @@ class CopyUrlButton extends Component {
...
+          )}
+        </InvisibleA11yLabel>
+        <Button className="tool-button" onClick={this._onClick}>
           {this.state.showCopied ? (
             <span>
               <IconWithText Icon={Check}>
```
Comment (miles-grant-ibigroup) [inline lib/components/narrative/trip-tools.js:68]: isn't this the same button again?

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Looks good, some nitpicks

File: lib/components/narrative/trip-tools.js (hunk 1 above)
Comment (amy-corson-ibigroup) [reply lib/components/narrative/trip-tools.js:62]: Actually, conditionally rendering the `aria-live` causes inconsistent announcements on AT. The `aria-live` `div` needs to be present even when it's empty, its like a tiny status region

File: lib/components/narrative/trip-tools.js (hunk 2 above)
Comment (amy-corson-ibigroup) [reply lib/components/narrative/trip-tools.js:68]: Yes the button itself hasn't changed just removing the `aria-live` element from the button itself and adding to the `div`!

File: lib/components/narrative/trip-tools.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/components/narrative/trip-tools.js:62]: Yikes that is UGLY! The a11y spec really has a ways to go...

File: lib/components/narrative/trip-tools.js (hunk 2 above)
Comment (miles-grant-ibigroup) [reply lib/components/narrative/trip-tools.js:68]: Yes but if you look at the file in the source code, it's there twice in a row!

Comment (miles-grant-ibigroup) [reply lib/components/narrative/trip-tools.js:68]: Never mind that's just us handling state poorly

File: lib/components/narrative/trip-tools.js (hunk 1 above)
Comment (amy-corson-ibigroup) [reply lib/components/narrative/trip-tools.js:62]: Long term we may want to look at creating a global status region that handles little announcements like this as well as the results from the geocoder, trip results, etc!!

**Review by daniel-heppner-ibigroup — APPROVED** [review]

**Pull request merged**